code/game/machinery/engi_points_vendor.dm

```diff
@@ -0,0 +1,183 @@
+
+/obj/machinery/engi_points_manager
+	name = "Engineering Points Manager"
+	desc = "Who's a good boy?"
+	icon = 'icons/obj/machines/engi_points.dmi'
+	icon_state = "store"
+	verb_say = "states"
+	density = 1
+	anchored = 1
+	var/obj/item/device/radio/radio
+	var/GBP = 0
+	var/GBPearned = 0
+	var/power_export_bonus = 0
+	var/air_alarm_bonus = 0
+	var/power_alarm_bonus = 0
+	var/fire_alarm_bonus = 0
+	var/alarm_rating = ""
+	var/prior_bonus = 2500
+	var/total_bonus = 0
+	var/GBP_alarm_cooldown = 4500
+	var/list/prize_list = list(

```

Comment:
Define lists at Init() or LAZYINITLIST, see https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#startupruntime-tradeoffs-with-lists-and-the-hidden-init-proc